auth-api/email_templates/govm_business_invitation_email.html

```diff
@@ -52,14 +52,15 @@
                 <tr>
                     <td>
                         <h1 style="margin: 0; line-height: 34px; letter-spacing: -1px; font-family: Verdana, Arial, sans-serif; font-size: 24px; font-weight: 700;">
-                            Your BC Registries account <strong>{{ org_name }}</strong> has been created.
+                            You've been invited to create BC Registries account.
                         </h1>
                         <p style="margin-top: 25px; margin-bottom: 0; line-height: 24px; font-family: Verdana, Arial, sans-serif; font-size: 16px;">
-                            BC Registries staff have approved your organization's account to search for minstry accounts.<br/> To complete the account setup, click the link below and create your Username and Password. Then login to your account and manage your team's access to Director Search.  </p>
+                            BC Registries staff have invited you to create an Business Registry account. You are the account administrator.<br/> To complete the account setup, click the button below and create your account. Then login to your account and manage your team members.
```

Comment:
I think;
`an Business Registry account`
should be
`a Business Registry account`

Reply:
will fix it.thanks